### PR TITLE
Scp 4606/e2e ls

### PR DESCRIPTION
**Pull request by ladamesny**
Implements basic end to end tests for marlowe run ls command.

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [x] [Test report is updated](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe/test/test-report.md) (if relevant)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested

### CODE REVIEW

File: marlowe-runtime/integration-tests/ls/non_empty_history.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+echo "Scenario: One contract is managed in history"
+echo "Expect to see $EXISTING_CONTRACT_ID in history"
+marlowe ls
+
+echo "Scenario: two contracts managed in history"
+marlowe add $NEW_CONTRACT_ID
+
+echo "Expect to see $EXISTING_CONTRACT_ID and $NEW_CONTRACT_ID in history"
+marlowe ls
```
Comment (bwbush) [inline marlowe-runtime/integration-tests/ls/non_empty_history.sh:34]: One could capture the output of this command to a variable or file, and then compare it to the expected value, returning a success or failure code.

File: marlowe-runtime/integration-tests/ls/empty_history.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+export CARDANO_NODE_SOCKET_PATH=/tmp/preview.socket
+export CARDANO_TESTNET_MAGIC=2
+MAGIC=(--testnet-magic 2)
+echo "${MAGIC[@]}"
+
+marlowe rm --all
+
+echo "Test Scnario set up done"
+
+marlowe ls
```
Comment (bwbush) [inline marlowe-runtime/integration-tests/ls/empty_history.sh:20]: One could capture the output of this command to a variable or file, and then compare it to the expected value, returning a success or failure code.

**Review by bwbush — CHANGES_REQUESTED** [review]
I think we could have this return error codes for success or failure. See comments.

**Comment by ladamesny** [comment]
Closing this PR as jupyter notebooks is not the approach we want to take for our integration tests.

**Pull request closed without merging**